src/JobHelper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace PrivatePackagist\ApiClient;
+
+use PrivatePackagist\ApiClient\Exception\JobErrorException;
+
+class JobHelper
+{
+    /** @var Client */
+    private $packagistClient;
+
+    public function __construct(Client $packagistClient)
+    {
+        $this->packagistClient = $packagistClient;
+    }
+
+    public function waitForJob($jobId, $maxWaitMinutes = 3)
```

Comment:
I think providing an option in seconds may be better if you potentially want to do this in a web context and just check if something returned quickly within 30s or so and otherwise provide other feedback?

Reply:
Changed